orix/tests/quaternion/test_orientation.py

```diff
@@ -57,7 +57,9 @@ def orientation(request):
     indirect=["orientation"],
 )
 def test_set_symmetry(orientation, symmetry, expected):
-    o = orientation.set_symmetry(symmetry)
+    o = Orientation(orientation.data)
+    o.symmetry = symmetry
```

Comment:
We should allow passing the symmetry upon initialization, no? This should be documented in the `Orientation.__init__(self, data, symmetry=None)` docstring. *C1* should be default when `symmetry` isn't given, to give same result as before.

```suggestion
    o = Orientation(orientation.data, symmetry)
```

Reply:
This should be done for `Misorientation.__init__(self, data, symmetry=None)` as well, with `symmetry` being a tuple with `Symmetry` instances.

Reply:
I suggest leaving this for another PR!

Reply:
I agree.

Reply:
Created issue #241 to discuss this.